docs/configGuide/YAML-Config-Syntax.md

```diff
@@ -557,6 +557,50 @@ We can observe that:
 * Additionally, if we set `blackouts` to `[-20]`, the arena will blackout every 20 frames (because we placed the '-' indicating repeat).
 * The blackout has no effect on any other aspect of the agent or the arena. For example, the agent will still be able to move around the arena, and the objects in the arena will still be visible to the agent. _RayCasting_ will still work. 
 
+#### EXAMPLE 8 - Multi-Arena Episodes
```

Comment:
Did you also update the TOC to add this new section?

Reply:
Potentially stupid question - what is TOC?

Reply:
table of contents in the md file :).